hasher-matcher-actioner/src/OpenMediaMatch/blueprints/matching.py

```diff
@@ -173,12 +173,33 @@ def lookup_threshold():
     Output:
      * List of matching with content_id, distance, and signal values
     """
-    signal = require_request_param("signal")
-    signal_type_name = require_request_param("signal_type")
+
+    if request.method == "POST":
+        if request.is_json:
+            params = request.get_json()
+        else:
+            params = request.form
+    else:  # GET
+        params = request.args
+
+    signal = params.get("signal")
```

Comment:
Maybe it's worth creating a method similar to `require_request_param` that also takes the params dict in order to match the previous style?